dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -871,6 +871,7 @@ def disabled?(selector)
 end
 
 Then(/^I slow down execution speed$/) do
+  @browser.execute_script("Maze.shouldSpeedUpInfiniteLoops = false;")
   @browser.execute_script("Maze.scale.stepSpeed = 10;")
```

Comment:
should we remove this line since we're setting `shouldSpeedUpInfiniteLoops`?

Reply:
I think we still want both. `scale.stepSpeed` is a multiplier to slow down each step. `shouldSpeedUpInfiniteLoops` being false just makes it so that the base speed isn't 0 so that the multiplier actually works